<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1576</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnRodar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Posicionar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblExibirIMG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aguardando Imagem para Exibição...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnManual">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>590</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="labelDX">
       <property name="text">
        <string>Delta X</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editDeltaX"/>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="labelDY">
       <property name="text">
        <string>Delta Y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editDeltaY"/>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="labelDZ">
       <property name="text">
        <string>Delta Z</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editDeltaZ"/>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnMover">
     <property name="text">
      <string>Mover</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnStereo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stereo</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>590</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="labelDA">
       <property name="text">
        <string>Delta A</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editDeltaA"/>
     </widget>
     <widget class="QSplitter" name="splitter_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="labelDB">
       <property name="text">
        <string>Delta B</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editDeltaB"/>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="labelDC">
       <property name="text">
        <string>Delta C</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editDeltaC"/>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnRotacionar">
     <property name="text">
      <string>Rotacionar</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnGAMAGON">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>GAMAG - ATIVAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGAMAGOFF">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>GAMAG - DESATIVAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1576</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
